doc/administrator_guide.md

```diff
@@ -94,6 +94,27 @@ Select the account key for use with Google Cloud API (which kcidb uses):
 Clone kcidb repository, checkout the revision you want to deploy, and install
 kcidb as described in [README.md](README.md).
 
+### Additional Dependencies
+
+While deploying you may also need to install `psql`. It is a command-line tool for
+interacting with the PostgreSQL database. It is used to manage and manipulate data
+stored in a PostgreSQL database, as well as to administer and configure the database itself.
+
+For Debian-based distros like Ubuntu:
+   
+    apt-get install postgresql-client
+
+For Fedora-based distros like Nobara:
```

Comment:
Hmm, never heard of Nobara, and never met anyone who used it. Could we drop this example? Otherwise it's confusing and distracting. The most popular Fedora-based distros are probably CentOS and RHEL, and we can mention them, perhaps.